packages/edit-site/src/components/page-templates/template-actions.js

```diff
@@ -24,6 +24,10 @@ import isTemplateRevertable from '../../utils/is-template-revertable';
 import isTemplateRemovable from '../../utils/is-template-removable';
 import { TEMPLATE_POST_TYPE } from '../../utils/constants';
 
+function hasThemeFile( template ) {
+	return !! template?.has_theme_file;
```

Comment:
`isTemplateRevertable` has one more check about the origin.

Reply:
The isTemplateRevertable is still present to check if the action is disabled or not. Here I'm checking if the action maybe be possible and is shown in the UI isTemplateRevertable will be used to check if it is enabled or disabled.